resources/views/profile.blade.php

```diff
@@ -0,0 +1,95 @@
+@extends('layouts.app')
+@section('content')
+    <div class="wrapper">
+        <div class="sidebar" data-color="green" >
+            @include('includes.sidebar')
+        </div>
+
+        <div class="main-panel">
+            <div class="container-fluid">
+                <div class="row">
+                    <div class="col-md-9">
+                        <h5>User Profile</h5>
+                        <hr>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-3 text-center">
+                        <img src="http://split.app/images/default-profile-picture.jpg" id="profileImage" style="width: 200px;">
+                        <h2>{{ Auth::user()->username }}</h2>
+                    </div>
+                    <div class="col-md-6">
+                        <table class="table table-striped table-bordered table-hover text-center control-label">
+                            <tr class="active">
+                                <td>First Name: </td>
+                                <td>{{ Auth::user()->firstname }}</td>
+                            </tr>
+                            <tr class="active">
+                                <td>Last Name: </td>
+                                <td>{{ Auth::user()->lastname }}</td>
+                            </tr>
+                            <tr class="active">
+                                <td>Email: </td>
+                                <td>{{ Auth::user()->email }}</td>
+                            </tr>
+                            <tr class="active">
+                                <td>Active expenses: </td>
+                                <td></td>
+                            </tr>
+                            <tr class="active">
+                                <td>Total expenses: </td>
+                                <td></td>
+                            </tr>
+                            <tr class="active">
+                                <td>Total friends: </td>
+                                <td></td>
+                            </tr>
+                        </table>
+
+                        <button href="#newExpense" class="btn btn-danger" data-toggle="collapse">Add expense for {{ Auth::user()->username }}</button>
```

Comment:
I see that the expense form is being used multiple times, could we separate this as different view file and just include it to the views where we need them?